app/config/ConfigKeys.scala

```diff
@@ -79,9 +79,12 @@ object ConfigKeys {
   val deregisterVatFrontendHost: String = "deregister-vat-frontend.host"
   val deregisterVatFrontendUrl: String = "deregister-vat-frontend.endpoints.deregister"
 
+  val vat7FormUrl: String = "external.vat7Form.url"
+  val vatHowToRegister: String = "external.vatHowToRegister.url"
+
```

Comment:
Normally agents hand off to agent services (https://www.gov.uk/guidance/vat-online-services-for-agents)

The criteria for the story did have the one you've used however I double checked with Martin and he's confirmed it should have the link for agent services. This is already in the repo however changes may be needed in a few places including tests